_observing-your-data/query-insights/live-queries.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: default
+title: Live queries
+parent: Query insights
```

Comment:
Should "Insights" be capitalized as in the preceding files?

Reply:
No because this refers to the concept. Open to updating if needed, but this is what we've decided early on.